### ISSUE TITLE
Add scaffolding defaults

### ISSUE THREAD

**Issue by bensheldon** [bensheldon/rails_new#1]
- ignore helpers, javascript, stylesheets

**Comment by bensheldon** [comment]
```ruby
Rails.application.config.generators do |g|
  g.test_framework :rspec, fixture: true, views: false, fixture_replacement: :factory_bot, view_specs: false
  g.fixture_replacement :factory_bot, dir: 'spec/factories'
  g.helper false
  g.assets false
  g.javascripts false
  g.stylesheets false
  g.view_specs false
  g.request_specs false
  g.routing_specs false
end
```

**Comment by bensheldon** [comment]
Closed by bdc767c73054f1 and 4422cbe97aefda62
